panel/src/components/Forms/Previews/EmailFieldPreview.vue

```diff
@@ -3,10 +3,6 @@ import UrlFieldPreview from "./UrlFieldPreview.vue";
 
 export default {
   extends: UrlFieldPreview,
-  computed: {
-    link() {
-      return this.value?.length > 0 ? "mailto:" + this.value : null;
```

Comment:
With this change, do we even need this component still?

Reply:
Good point. Probably only for the class, but we could at least remove the computed link

Reply:
We still need it to keep the correct class name. It could be a breaking change if someone styled it differently.